server/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:20.13.1-alpine
+
+WORKDIR /usr/src/app
+
+RUN npm install -g npm@10.2.4
+
+COPY package*.json ./
+
+COPY . .
```

Comment:
Could you please move the COPY instructions just before the EXPOSE instruction? This will help optimize the build process.